__tests__/app.test.js

```diff
@@ -49,6 +49,7 @@ describe('GET /api/articles/:article_id', () => {
             body: "I find this existence challenging",
             created_at: "2020-07-09T20:11:00.000Z",
             votes: 100,
+            comment_count: '11'
```

Comment:
You should not edit a previous test to add this key. If this test fails because you've added a new key, it was not a robust enough test to begin with. 

There should be a completely new test to check for the comment_count key.